GUI tests are failing intermittently

The GUI tests are fialing intermittently due to a timeout (see below). My guess is that raising the timeout threshold by a bit could solve this issue.

```
Run pytest test_gui.py
============================= test session starts ==============================
platform linux -- Python 3.10.16, pytest-8.3.5, pluggy-1.5.0
rootdir: /home/runner/work/streamlit-template/streamlit-template
collected 27 items

test_gui.py ..................F........                                  [100%]

=================================== FAILURES ===================================
_________ test_view_raw_ms_data[content/raw_data_viewer.py-Blank.mzML] _________

launch = AppTest(
    _script_path='content/raw_data_viewer.py',
    default_timeout=3,
    session_state={$$ID-8771a678d5dc1d0...$$ID-e18cf3d2836df333c1c5784f084e2e2c-None: False},
    secrets={
        workspace: 'test'
    },
    _tree={

    }
)
example = 'Blank.mzML'

    @pytest.mark.parametrize('launch,example', [("content/raw_data_viewer.py", 'Blank.mzML'),
                                                ("content/raw_data_viewer.py", 'Treatment.mzML'),
                                                ("content/raw_data_viewer.py", 'Pool.mzML'),
                                                ("content/raw_data_viewer.py", 'Control.mzML')], indirect=['launch'])
    def test_view_raw_ms_data(launch, example):
>       launch.run()

test_gui.py:69: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
/opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/streamlit/testing/v1/app_test.py:391: in run
    return self._tree.run(timeout=timeout)
/opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/streamlit/testing/v1/element_tree.py:1924: in run
    return self._runner._run(widget_states, timeout=timeout)
/opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/streamlit/testing/v1/app_test.py:350: in _run
    self._tree = script_runner.run(
/opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/streamlit/testing/v1/local_script_runner.py:128: in run
    require_widgets_deltas(self, timeout)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

runner = LocalScriptRunner(forward_msg_queue=<streamlit.runtime.forward_msg_queue.ForwardMsgQueue object at 0x7f4aa476d270>, sc...ue
}
}, {}, {'client_state': query_string: "workspace=default"
page_script_hash: "43cd02e93e7accf3cb50f363a64f5f6f"
}])
timeout = 3

    def require_widgets_deltas(runner: LocalScriptRunner, timeout: float = 3) -> None:
        """Wait for the given ScriptRunner to emit a completion event. If the timeout
        is reached, the runner will be shutdown and an error will be thrown.
        """
    
        t0 = time.time()
        while time.time() - t0 < timeout:
            time.sleep(0.001)
            if runner.script_stopped():
                return
    
        # If we get here, the runner hasn't yet completed before our
        # timeout. Create an error string for debugging.
        err_string = f"AppTest script run timed out after {timeout}(s)"
    
        # Shutdown the runner before throwing an error, so that the script
        # doesn't hang forever.
        runner.request_stop()
        runner.join()
    
>       raise RuntimeError(err_string)
E       RuntimeError: AppTest script run timed out after 3(s)

/opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/streamlit/testing/v1/local_script_runner.py:180: RuntimeError
---------------------------- Captured stderr setup -----------------------------
2025-03-07 09:58:33.924 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.
2025-03-07 09:58:33.924 Thread 'MainThread': missing ScriptRunContext! This warning can be ignored when running in bare mode.
=============================== warnings summary ===============================
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Treatment.mzML]
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Treatment.mzML]
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Pool.mzML]
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Pool.mzML]
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Control.mzML]
test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Control.mzML]
  /opt/hostedtoolcache/Python/3.10.16/x64/lib/python3.10/site-packages/pyopenms_viz/_misc.py:347: UserWarning:
  
  auto computed (1pct-diff) tolerance is 0. Using default tolerance value of 1

-- Docs: https://docs.pytest.org/en/stable/how-to/capture-warnings.html
=========================== short test summary info ============================
FAILED test_gui.py::test_view_raw_ms_data[content/raw_data_viewer.py-Blank.mzML] - RuntimeError: AppTest script run timed out after 3(s)
================== 1 failed, 26 passed, 6 warnings in 44.25s ===================
```

--- a/test_gui.py
+++ b/test_gui.py
@@ -66,7 +66,7 @@ def test_file_upload_load_example(launch):
                                             ("content/raw_data_viewer.py", 'Pool.mzML'),
                                             ("content/raw_data_viewer.py", 'Control.mzML')], indirect=['launch'])
 def test_view_raw_ms_data(launch, example):
-    launch.run()
+    launch.run(timeout=10)
 
     ## Load Example file, based on implementation of fileupload.load_example_mzML_files() ###
     mzML_dir = Path(launch.session_state.workspace, "mzML-files")

--- a/test_gui.py
+++ b/test_gui.py
@@ -66,7 +66,7 @@ def test_file_upload_load_example(launch):
                                             ("content/raw_data_viewer.py", 'Pool.mzML'),
                                             ("content/raw_data_viewer.py", 'Control.mzML')], indirect=['launch'])
 def test_view_raw_ms_data(launch, example):
-    launch.run(timeout=10)
+    launch.run()
 
     ## Load Example file, based on implementation of fileupload.load_example_mzML_files() ###
     mzML_dir = Path(launch.session_state.workspace, "mzML-files")

